artifacts/testdata/windows/volatile.in.yaml

```diff
@@ -0,0 +1,8 @@
+Queries:
+  - SELECT Name, Path, CommandLine, 
+      Hash, Username, Authenticode, 
+      Family, Type, Status,
+      Laddr.IP, Laddr.Port,
+      Raddr.IP, Raddr.Port
+    FROM Artifact.Windows.Network.NetstatEnriched()
+    WHERE Name = nc.exe
```

Comment:
I think this should be Name = "nc.exe"

Reply:
done.

init.cmd should of also been: start cmd /c "artifacts\testdata\files\nc.exe -L -p 3889 -s 0.0.0.0"